params/types/genesisT/genesis.go

```diff
@@ -520,6 +520,14 @@ func (g *Genesis) SetECIP1080Transition(n *uint64) error {
 	return g.Config.SetECIP1080Transition(n)
 }
 
+func (g Genesis) GetEIP1706Transition() *uint64 {
+	return g.Config.GetECIP1080Transition()
```

Comment:
Wrong ECIP/EIP number ;)
```suggestion
	return g.Config.GetEIP1706Transition()
```

Reply:
Resolved with https://github.com/etclabscore/multi-geth/pull/126/commits/c51dc1efa03629f5156e3fd8be1983b107d788c9